src/vaadin-overlay.html

```diff
@@ -286,12 +291,16 @@
 
           /**
            * References the content container after the template is stamped.
+           * @type {HTMLElement}
            */
           content: {
             type: Object,
             notify: true
           },
 
+          /**
+           * When true the overlay has backdrop on top of content when opened.
+           */
           withBackdrop: {
             type: Boolean,
```

Comment:
The current result is:
```ts
withBackdrop: boolean|null|undefined;
```

I would suggest:
```suggestion
          /**
           * When true the overlay has backdrop on top of content when opened.
           * @type {boolean}
           */
          withBackdrop: {
            type: Boolean,
```
to get:
```ts
withBackdrop: boolean;
```
since this also has a default value of `false`.

Reply:
Please see my comment https://github.com/vaadin/vaadin-overlay/pull/179#discussion_r419284855

Reply:
Added `@type {boolean}`.